src/tests/test_ucc_verifier.cpp

```diff
@@ -0,0 +1,153 @@
+#include <gtest/gtest.h>
+
+#include "algorithms/algo_factory.h"
+#include "algorithms/ucc/ucc_verifier/ucc_verifier.h"
+#include "config/indices/type.h"
+#include "datasets.h"
+
+namespace tests {
+namespace onam = config::names;
+
+class UCCVerifierSimpleParams {
```

Comment:
Move all classes definitions which are required only in this file to the anonymous namespace